monkestation/code/modules/hydroponics/machines/composter.dm

```diff
@@ -81,6 +87,15 @@
 	else
 		. += mutable_appearance('monkestation/icons/obj/machines/composter.dmi', "light_on", layer = OBJ_LAYER + 0.01)
 
+/obj/machinery/composters/screwdriver_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if(!.)
+		return default_deconstruction_screwdriver(user, "composter_open", "composter", tool)
```

Comment:
is this behavior not normally a thing on machines?

Reply:
doesn't seem like it